app/Models/Playlist.php

```diff
@@ -18,8 +22,37 @@ class Playlist extends Model
         'title',
         'project_id',
         'order',
+        'shared'
```

Comment:
I could not find migration for adding this field to the table.

Reply:
Mahmood is working on it, 
he will be adding migrations for "is_public" field as well in his pull request.